### ISSUE TITLE
Nuget Manager -The plugin credential provider could not acquire credentials. Authentication may require manual action

### ISSUE THREAD

**Issue by sureshbm** [microsoft/BotFramework-Composer#9632]
## Describe the bug

I am getting the below error while trying to create a new bot using Bot Framework Composer.

[NuGet Manager] [Warning] The plugin credential provider could not acquire credentials. Authentication may require manual action. Consider re-running the command with --interactive for `dotnet`, /p:NuGetInteractive="true" for MSBuild or removing the -NonInteractive switch for `NuGet`

## Version

V2.1.2

## Browser

- [ ] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [X ] Windows
- [ ] Ubuntu

## To Reproduce

1. Start Bot Framework Composer Application.
2. Click on 'Create New
3. Select  'Empty BO'

## Expected behavior

Empty Bot application should be created.
## Screenshots

![image](https://github.com/microsoft/BotFramework-Composer/assets/19537330/0f78f075-ed12-4432-8129-45d48ad3c8be)

## Additional context

Installed .NET Framework 4.8 Developer Pack  
Installed .NET Core 3.1 SDK
Visual Studio 2019 and 2022 both are installed.
Node JS installed

**Comment by ramfattah** [comment]
Thanks @sureshbm, we are investigating this issue.

**Comment by JaneIllario** [comment]
Not a fix, but a workaround -- I had this error with Node.js 18.17.1 (includes npm 9.6.7), and it got resolved with an update to Latest Current Version: 20.6.1 (includes npm 9.8.1)

**Comment by ramfattah** [comment]
Hi @sureshbm, 

It appears I'm not able to reproduce this issue:

https://github.com/microsoft/BotFramework-Composer/assets/38049078/6e008830-7ec8-4a17-8232-eb1516ce27c6

Environment:

```
Operating System: macOS M1 Ventura 13.4
```

``` zsh
# dotnet --list-sdks
6.0.410 [/usr/local/share/dotnet/sdk]
6.0.412 [/usr/local/share/dotnet/sdk]
6.0.413 [/usr/local/share/dotnet/sdk]
7.0.304 [/usr/local/share/dotnet/sdk]
7.0.306 [/usr/local/share/dotnet/sdk]
7.0.307 [/usr/local/share/dotnet/sdk]
7.0.400 [/usr/local/share/dotnet/sdk]
8.0.100-preview.6.23330.14 [/usr/local/share/dotnet/sdk]
```

```zsh
# npm --version
8.19.4
```

```zsh
# node --version
v16.20.1
```


#

Asking a few clarifying questions:
1. Could you please confirm if you are using a `C#` or `NodeJS` Composer bot? 
2. Are you using any private NuGet feeds or repositories for your project? If yes, can you confirm if they require authentication?
3. Do you have multiple NuGet sources or feeds configured for your project? Are they a mix of public (like NuGet.org) and private feeds?
4. Do you have a `nuget.config` file in your project or solution directory? This file contains NuGet settings and source configurations.

**Comment by ramfattah** [comment]
Hi @sureshbm, any status update on this? 

Have you tried @JaneIllario's workaround to see if that address your issue?

**Comment by ramfattah** [comment]
Closing due to inactivity. @sureshbm, please re-open if needed.

**Comment by JanEggers** [comment]
same here after updateing to vs 2022 17.8

**Comment by danneforslund** [comment]
> same here after updateing to vs 2022 17.8.1

Same here.

**Comment by n-athan** [comment]
> On prem buildserver with VS 2022 17.8 connected to Azure Devops Services Build fails with
> 
> ##[warning]D:\B\A01_tool\dotnet\sdk\8.0.100\NuGet.targets(156,5): Warning : The plugin credential provider could not acquire credentials. Authentication may require manual action. Consider re-running the command with --interactive for `dotnet`, /p:NuGetInteractive="true" for MSBuild or removing the -NonInteractive switch for `NuGet`

I solved this by adding the NugetAuthenticate@1 task to my pipeline.

**Comment by erikthysell** [comment]
> > same here after updateing to vs 2022 17.8.1
> 
> Same here.

same here...

**Comment by malthuswaswrong** [comment]
This is not a closed issue. It is happening with Azure DevOps pipelines using MSBuild task to build Azure Function projects using Visual Studio Version 17.12.0 Preview 2.1.

The pipeline would build all the other projects but would crash building Azure Functions with "The plugin credential provider could not acquire credentials" when trying to access the organization's private ADO repo.

The workaround proposed by @n-athan seems to be holding for me.  I added NugetAuthenticate@1 with no parameters.  When I kicked off the pipeline manually it halted asking for more permissions.  I clicked the button to grant permission and then the pipeline continued without error and successfully built the Azure Function projects.
